cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,149 @@
 /// <reference types='cypress' />
 
 describe('Web Tables page', () => {
-  it('', () => {
+  let user;
 
+  beforeEach(() => {
+    cy.task('generateUser').then((generateUse) => {
+      user = generateUser;
+    });
+
+    cy.visit('/webtables');
+  });
+
+  it('Pagination', () => {
+    cy.get('.-previous')
+      .should('contain', 'Previous');
+    cy.get('.-pageInfo')
+      .should('exist');
+    cy.get('.-next')
+      .should('contain', 'Next');
+  });
+
+  it('Rows count selection', () => {
+    cy.get('select')
+      .should('exist')
+      .should('be.visible');
+  });
+
+  it('Add a new worker', () => {
+    cy.get('#addNewRecordButton')
+      .click();
+    cy.get('#firstName')
+      .type(user.firstName);
+    cy.get('#lastName')
+      .type(user.lastName);
+    cy.get('#userEmail')
+      .type(user.email);
+    cy.get('#age')
+      .type(user.age);
+    cy.get('#salary')
+      .type(user.salary);
+    cy.get('#department')
+      .type(user.department);
+    cy.get('#submit')
+      .click();
+    cy.get('.rt-td')
+      .should('contain', user.firstName)
+      .should('contain', user.lastName)
+      .should('contain', user.email)
+      .should('contain', user.salary)
+      .should('contain', user.department)
+      .should('contain', user.age);
+  });
+
+  it('Delete a worker', () => {
+    cy.get('#delete-record-1')
+      .click();
+    cy.get('.rt-table')
+      .should('not.contain', '#delete-record-1');
+  });
+
+  it('Delete all workers', () => {
+    cy.get('#delete-record-1')
+      .click();
+    cy.get('#delete-record-2')
+      .click();
+    cy.get('#delete-record-3')
+      .click();
+    cy.get('.rt-table')
+      .should('not.contain', '#delete-record-1')
+      .should('not.contain', '#delete-record-2')
+      .should('not.contain', '#delete-record-3');
+  });
+
+  it('Find a worker in the search field and edit it', () => {
+    cy.get('#searchBox')
```

Comment:
Hard-coded value 'Alden' is used in test case 'Find a worker in the search field and edit it'. Consider using variables for such values, so that they can be easily changed or randomized for further testing.